src/sushi/lib/contracts.js

```diff
@@ -64,7 +64,7 @@ export class Contracts {
       poolAddress,
       poolContract,
       name: `${token0Symbol}-${token1Symbol} Pool`,
-      symbol: `${token0Symbol}-${token1Symbol} UNI-V2 LP`,
+      symbol: `${token0Symbol}-${token1Symbol} HNY-V2 LP`,
```

Comment:
Did we change this for a reason? The actual symbol of the honeyswap pairs is UNI-V2 LP.

Reply:
We talked about that it's confusing that the LP tokens are named after uniswap. However, yeah, the fact that this name is encoded in the contracts makes sense. I will revert the change.